src/envinfo.js

```diff
@@ -19,7 +19,7 @@ module.exports.helpers = helpers;
 module.exports.envinfo = function print(options) {
   const props = {
     System: ['OS', 'CPU', 'Free Memory', 'Total Memory', 'Shell'],
-    Binaries: ['Node', 'Yarn', 'npm', 'Watchman', 'Docker', 'Homebrew'],
+    Binaries: ['Node', 'Yarn', 'npm', 'Watchman', 'Docker', 'Homebrew', 'Android SDKs'],
```

Comment:
I think we should probably create a new top level for this.

Reply:
What do you want to name it?

Reply:
Just SDKs sounds good to me.

Reply:
yeah, this way we can add iOS in the future.